citizen/quick_start_ICLI.md

```diff
@@ -0,0 +1,75 @@
+[ ICON CLI tool 설치 및 사용법 ]
```

Comment:
Need to consider this document translated
(project heath check also needed: https://github.com/icon-project/icon_cli_tool)

Reply:
It will be removed.